Jenkinsfile

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env groovy
+
+node('rhel7'){
+	stage('Checkout repo') {
+		deleteDir()
+		git url: 'https://github.com/redhat-developer/openshift-vsts.git',
+			branch: "${BRANCH}"
+	}
+
+	stage('Install requirements') {
+		def nodeHome = tool 'nodejs-8.11.1'
+		env.PATH="${env.PATH}:${nodeHome}/bin"
+		sh "npm run setup"
+	}
+
+	stage('Build') {
+		sh "npm install"
+		sh "npm run build"
+	}
+
+	withEnv(['JUNIT_REPORT_PATH=report.xml']) {
+        stage('Test') {
+    		wrap([$class: 'Xvnc']) {
+    			sh "npm run test:report:ci"
+    			junit '**/test-report.xml'
+    		}
+        }
+	}
+
+	stage('Package') {
+        sh "npm run extension:create:dev"
```

Comment:
@jeffmaury i only have one doubt here. It seems to me that the package is only created by using "npm run extension:create:dev" and then it is published in the marketplace. is that correct? If so we need to change it to "npm run extension:create". If we use extension:create:dev, the vsix-manifest-overrides.json will be used during the package phase which contains "public": "false" setting and the extension will be no visible to the marketplace. This setting is used for testing, so you can create your extension and test it without make it visible to others.

Reply:
Good catch. I copied from azure pipeline but the concern is probably different